Dockerfile

```diff
@@ -32,6 +32,7 @@ COPY --from=intermediate /out /out
 RUN cd / && tar zxvf /out/sharkd.tar.gz && rm -rf /out/sharkd.tar.gz
 
 ENV CAPTURES_PATH=/captures/
+ENV SHARKD_SOCKET=/home/node/sharkd.sock
```

Comment:
I've had a good degree of success placing the SHARKD_SOCKET within the CAPTURES_PATH since its pretty much guaranteed to be available, writeable and exclusive regardless of the container build or username. thoughts?